src/Connection.php

```diff
@@ -352,11 +358,9 @@ public function logQueries(bool $logQueries = true) : void
     {
         $this->logQueries = $logQueries;
 
-        $statementClass = ($this->logQueries)
+        $this->statementClass = ($this->logQueries)
             ? LoggedStatement::CLASS
             : PDOStatement::CLASS;
-
-        $this->pdo->setAttribute(PDO::ATTR_STATEMENT_CLASS, [$statementClass]);
```

Comment:
@dbraff Why remove this setAttribute() call?

Reply:
This is what breaks when persistent connections are enabled. PDO will not let you both have that option enabled and also set the statement class in this way. We could theoretically make that conditional based on the whether or not persistent connections have been enabled, but I thought that once the `LoggedStatement` had been modified to support this, it would simpler to leave out that type of conditional logic.

Reply:
Ah, never mind: "PDO::ATTR_STATEMENT_CLASS: Set user-supplied statement class derived from PDOStatement. Cannot be used with persistent PDO instances."